GOVERNANCE.md

```diff
@@ -61,6 +61,8 @@ Maintainer on receipt of a security issue or CoC report.  All current Maintainer
 must be invited to such closed meetings, except for any Maintainer who is
 accused of a CoC violation.
 
+More details can be found [here](commnuity_meeting.md).
```

Comment:
Can you move this above the closed meeting paragraph?

Reply:
nit - You forgot to remove the original link here.

Reply:
Hm. I think I missed to push a change. Done.